keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -806,73 +775,40 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason:
-- [ ] API .status
-  - Condition name:
-  - Other field:
-- [ ] Other (treat as last resort)
-  - Details:
+- [x] Events
+  - Event Reason: KubernetesDefaultServiceCIDRError, "The default ServiceCIDR can not be created"
+  - Event Reason: KubernetesDefaultServiceCIDRError, "The default ServiceCIDR Status can not be set to Ready=True"
+  - Event Reason: ClusterIPOutOfRange, ClusterIP is not within any configured Service CIDR; please recreate service
+  - Event Reason: ClusterIPNotAllocated, ClusterIP is not allocated; repairing
+  - Event Reason: UnknownError, Unable to allocate Cluster IP due to unknown error
+  - Event Reason: ClusterIPNotAllocated, ClusterIP has a wrong reference; repairing
+  - Event Reason: ClusterIPAlreadyAllocated, ClusterIP was assigned to multiple services; please recreate service
+  - Event Reason: IPAddressNotAllocated, IPAddress appears to have been modified, not referencing a Service : cleaning up
+  - Event Reason: IPAddressNotAllocated, IPAddress appears to have leaked: cleaning up
+  - Event Reason: IPAddressWrongReference, IPAddress for Service has a wrong reference; cleaning up
+
+- [x] API .status
+  - Condition name: ServiceCIDRConditionReady, represents status of a ServiceCIDR that is ready to be used by the apiserver to allocate ClusterIPs for Services.
+  - Condition name: ServiceCIDRReasonTerminating, represents a reason where a ServiceCIDR is not ready because it is being deleted.
```

Comment:
```suggestion
  - Condition reason: ServiceCIDRReasonTerminating, represents a reason where a ServiceCIDR is not ready because it is being deleted.
```